www/components/useMonaco.ts

```diff
@@ -1,9 +1,9 @@
 // ported from https://github.com/suren-atoyan/monaco-react
 
-import { useState } from 'preact/hooks';
+import { useState } from "preact/hooks";
```

Comment:
Would be kinda cool to use preact signals, but might be hard considering this originally isn't our code.

Reply:
oh yeah had to port it directly but it works